emails/tests/utils_tests.py

```diff
@@ -91,6 +92,23 @@ def test_generate_relay_From_with_premium_user(self):
         )
         assert formatted_from_address == expected_formatted_from
 
+    def test_generate_relay_From_with_new_from_user(self):
```

Comment:
😄 I love tests for new functionality! And bonus, the old functionality was already covered by tests!